styles/filter.css

```diff
@@ -0,0 +1,62 @@
+.filter-container {
+    background-color: var(--tertiary-color);
+    width: 15rem;
+}
+
+.items {
+    padding: 1rem;
+}
+
+.filter {
+    display: flex;
+    justify-content: space-between;
+}
+
+.filter-text {
+    font-size: 1.2rem;
+    font-weight: 700;
+    margin: 1rem 0;
+}
+
+.clear-text {
+    border: none;
+}
+
+.clear-text:hover {
+    color: var(--secondary-color);
+}
+
+.inputs {
+    font-size: 1rem;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+}
+
+.radio-unchecked {
+    margin: 0.4rem;
```

Comment:
values less than 1rem can be written in  px (upto 10px)

Reply:
Yes. Thanks for suggestion.